TheWarWithin/DruidFeral.lua

```diff
@@ -1461,16 +1461,19 @@ end ) ]]
 
 spec:RegisterStateFunction( "check_bloodtalons", function ()
     if buff.bt_triggers.stack > 2 then
-        -- Remove each individual BT buff.
-        for _, k in ipairs( buff.bt_triggers.alias ) do
+
+        local bt_list = buff.bt_triggers.alias or {}
```

Comment:
I'd rather not create tables in the loop if not needed.  Can just encapsulate the ipairs loop in an `if buff.bt_triggers.alias then` -- if you tried this an `buff.bt_triggers.alias` was returning 0, you can just do `if type( buff.bt_triggers.alias ) == "table" then` instead.